va/caa.go

```diff
@@ -245,8 +245,9 @@ func (va *ValidationAuthorityImpl) performRemoteCAACheck(
 	for _, i := range rand.Perm(len(va.remoteVAs)) {
 		remoteVA := va.remoteVAs[i]
 		go func(rva RemoteVA) {
+			perspective := rva.Perspective
 			result := &remoteVAResult{
-				VAHostname: rva.Address,
```

Comment:
This was removed, if you add it back as I have suggested in other comments it should be `Address` as this is the [IP Address]:Port of the remote VA.